infrastructure/nginx_config.conf

```diff
@@ -2,9 +2,9 @@ server {
   listen 8080;
   root /etc/nginx/html;
 
-  location /api/v1/workflow {
-    proxy_pass http://workflow-manager:8000/api/v1/workflow;
-  }
+  # location /api/v1/workflow {
+  #   proxy_pass http://workflow-manager:8000/api/v1/workflow;
+  # }
```

Comment:
is this commented out on purpose?

Reply:
Thank you! Forgot abou this, it was just for development purposes